proto/feature/clause.proto

```diff
@@ -32,6 +32,7 @@ message Clause {
     AFTER = 10;
     // Attribute is feature ID, and value is variation ID.
     FEATURE_FLAG = 11;
+    PARTIALLY_EQUALS = 12;
```

Comment:
Thank you so much for your contribution!

The `PARTIALLY_MATCH` is a better fit for this case.
Could you update the other places?

```suggestion
    PARTIALLY_MATCH = 12;
```

Reply:
@cre8ivejp 
I recreated the commit to have `PARTIALLY_MATCH`.